### ISSUE TITLE
Any plans on iresnet?

### ISSUE THREAD

**Issue by gitlabspy** [leondgarse/Keras_insightface#56]
Thx for sharing your works!
Any plans on making pretrained iresnet backbone (the one used in official arcface) ?

**Comment by leondgarse** [comment]
The `r50 swish` model shared in [readme/accuracy](https://github.com/leondgarse/Keras_insightface#current-accuracy) is similar with `iresnet`. Just using `swish` instead of `PReLU`. No further plan training the exactly same one using `PReLU`, as the `r50` model on `Glint360K` is good enough.

**Comment by gitlabspy** [comment]
Thanks!
